.travis.yml

```diff
@@ -1,57 +1,36 @@
-sudo: true
-
+sudo: required
 language: node_js
-
 os:
-  - osx
-  - linux
-
-sudo: required
+- osx
+- linux
 dist: trusty
-
 osx_image: xcode9.0
-
 node_js:
-  - 8
-
-branches:
-  only:
-  - master
-  - /^v\d+\.\d+(\.\d+)?(-\S*)?$/
-
+- 8
 cache:
   yarn: true
   directories:
-    - node_modules
-    - app/node_modules
-
+  - node_modules
+  - app/node_modules
 addons:
   apt:
     sources:
-      - ubuntu-toolchain-r-test
+    - ubuntu-toolchain-r-test
     packages:
-      - g++-4.8
-      - icnsutils
-      - graphicsmagick
-      - xz-utils
-      - xorriso
-
-#install:
-#  - export CXX="g++-4.8"
-#  - yarn
-#  - cd app && yarn && cd ..
-#  - "/sbin/start-stop-daemon --start --quiet --pidfile /tmp/custom_xvfb_99.pid --make-pidfile --background --exec /usr/bin/Xvfb -- :99 -ac -screen 0 1280x1024x16"
-
-#before_script:
-#  - export DISPLAY=:99.0
-#  - sh -e /etc/init.d/xvfb start &
-#  - sleep 3
-
+    - g++-4.8
+    - icnsutils
+    - graphicsmagick
+    - xz-utils
+    - xorriso
+before_install:
+  - echo "@zen:registry=https://www.myget.org/F/zenprotocol/npm/" > .npmrc
+  - echo "//www.myget.org/F/zenprotocol/npm/:_authToken=$MYGET_TOKEN" >> .npmrc
+install:
+  - npm i -g npm@latest
```

Comment:
yeah, I wanted to be sure that it install the latest version from npm

Reply:
Did it fail without it? Ideally we want to make the builds pass as fast as possible, because when we're dev'ing, we don't want to wait long for PRs checks.

i would try without explicitly installing npm latest, and if it fails, try some solutions like in [this issue](https://github.com/travis-ci/travis-ci/issues/4653)